libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/DropInNavigationView.kt

```diff
@@ -66,6 +67,16 @@ class DropInNavigationView @JvmOverloads constructor(
         viewModel = viewModel,
     )
 
+    var isReplayEnabled: Boolean
```

Comment:
What is the reason this is here? seems like it should be with all the other state

Reply:
Yes. 
Adding `isReplayEnabled ` property was necessary for `MapboxNavigationViewCustomizedActivity` to be able to read and update `TripSessionStarterState.isReplayEnabled` that is buried deep inside the Store which is internal.

Reply:
https://github.com/mapbox/mapbox-navigation-android/pull/5626/files#diff-1181d521b74e76ffaacfed0d26a9c59920179338a76b4c61d1b0a7ad46437ca3R145

There is a TODO above that `MapboxNavigationViewCustomizedActivity` to create a better interface for surfacing state in `DropInNavigationView`. The current solution is dirty, but I don't think this change is solving it

We should not surface the `Store` state like this inside DropInNavigationView.